### DISCUSSION TITLE
uploaded app-url in `upload_to_testflight`

### DISCUSSION THREAD

**Discussion by aap01** [fastlane/fastlane#21439] [Q&A]
How do I get the uploaded `app url` after uploading it with the `upload_to_testflight` action?

I am looking for a link to share with my internal tester group so that they can click to visit the latest build in TestFlight.

Here is my code

```ruby
    release = upload_to_testflight(
      app_version: new_version,
      build_number: new_build_number,
      api_key: appstore_api_key,
      ipa: atrtifact_path,
      changelog: release_notes
    )
    link = lane_context[SharedValues::UPLOAD_TO_TESTFLIGHT_APP_LINK]
```

**Comment by enes-ozdemir** [comment]
Hey 👋,
Any luck finding the answer ?
